charts/prometheus-blackbox-exporter/templates/networkpolicy.yaml

```
@@ -15,7 +15,7 @@ spec:
   - from:
     - namespaceSelector:
         matchLabels:
-          name: monitoring
+          name: {{ default "monitoring" .Values.networkPolicy.monitoringNamespaceName }}
```

Any reason not to use: `Release.Namespace`?

```suggestion
          name: {{ .Values.networkPolicy.monitoringNamespaceName }}
```